.nvmrc

```diff
@@ -1 +1 @@
-v16.0.0
\ No newline at end of file
+v16.1.0
```

Comment:
Have you tried upgrading to v16.15.1?  This will keep the node version same across extensions and language server.